src/lib/components/data-graphic/guides/Axis.svelte

```diff
@@ -173,12 +174,18 @@ This component will draw an axis on the specified side.
     // use graphicWidth or graphicHeight
     // do we ensure different spacing in one case vs. another?
     tickCount = ~~(axisLength / 20);
-    tickCount = Math.max(2, ~~(tickCount / 100));
+    const maxTickCountThreshold = 2 + ~~($plotConfig.width / 250);
```

Comment:
I think we should be using `axisLength` not `$plotConfig.width` here. Otherwise the same truncation doesn't seem to happen with the y axis. This probably explains the buggy GIF I posted in another comment.

![rill-developer-axis-bug](https://user-images.githubusercontent.com/95735/180797906-d5181cd5-d87c-49b0-99e3-72853e091b36.gif)